### ISSUE TITLE
Retrieving the similarity values

### ISSUE THREAD

**Issue by ttumkaya** [jokergoo/simplifyEnrichment#65]
Hi @jokergoo,

Thanks for this very useful package! 

I was wondering if there is an easy way of retrieving the similarity index between GO terms, or even better the "Block mean" values that is described in the vignette?

Best,
Tayfun

**Comment by jokergoo** [comment]
Yes, there is an internal function `simplifyEnrichment:::block_mean()`. You can do like:

```r
cl = binary_cut(mat)
simplifyEnrichment:::block_mean(mat, cl)
```
